src/js/common/reducers/index.js

```diff
@@ -1,10 +1,45 @@
 import { combineReducers } from 'redux';
 
-import login from '../../common/reducers/login';
-import profile from '../../common/reducers/profile';
+import login from '../../common/reducers/user/login';
+import profile from '../../common/reducers/user/profile';
+
+import alertRx from '../../common/reducers/rx/alert';
+import disclaimer from '../../common/reducers/rx/disclaimer';
+import modalsRx from '../../common/reducers/rx/modals';
+import prescriptions from '../../common/reducers/rx/prescriptions';
+
+import alert from '../../common/reducers/messaging/alert';
+import compose from '../../common/reducers/messaging/compose';
+import folders from '../../common/reducers/messaging/folders';
+import loading from '../../common/reducers/messaging/loading';
+import messages from '../../common/reducers/messaging/messages';
+import modals from '../../common/reducers/messaging/modals';
+import recipients from '../../common/reducers/messaging/recipients';
+import search from '../../common/reducers/messaging/search';
```

Comment:
If we're keeping all the other apps' reducers under `common/reducers`, these paths won't need the `../../common/reducers/` in front.